lib/cli/src/commands/create_exe.rs

```diff
@@ -813,22 +820,22 @@ impl CreateExe {
                         let mut include_dir = libwasmer_path.clone();
                         include_dir.pop();
                         include_dir.push("include");
-                        println!("include dir: {}", include_dir.display());
                         let mut cmd = Command::new(zig_binary_path);
                         let mut cmd_mut: &mut Command = cmd
                             .arg("cc")
                             .arg("--verbose")
                             .arg("-target")
                             .arg(&zig_triple)
-                            .arg(&format!("-L{}", libwasmer_path.display()))
-                            .arg(&format!("-l:{}", lib_filename))
-                            .arg(&format!("-I{}", include_dir.display()));
+                            .arg(&format!("-I{}", include_dir.display()))
+                            .arg("-lc");
```

Comment:
Do we need to do the `if windows { cmd.arg("-lc++") } else { cmd.arg("-lc") }` thing here, too?

Reply:
I've refactored this so we don't need this code block at all anymore.

Also this code block is impossible to hit because cross-compilation + ObjectFormat::Serialized doesn't work in wasmer.